recipes/mudirac/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "mudirac" %}
+{% set version = "1.0.1" %}
+
+
+package:
+  name: mudirac
+  version: 1.0.1
+
+source:
+  url:  https://github.com/muon-spectroscopy-computational-project/mudirac/archive/refs/tags/1.0.1.tar.gz
+  sha256: 801cf46c71726265f988e984f539354c6fbab070dc101090009ee92d09ce0faa
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
```

Comment:
Why are they failing? Seems like a simple recipe and this it would be nice to include them.

Reply:
now not skipping for osx, which works fine. As the mudirac package is using `make` build it is not building against windows. Modifying the build system suitable for windows can be considered in the future.

Reply:
tried osx, but failed due to one of the library dependencies https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=916285&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1414

Reply:
will conder OSx in in the future releases.